zds/gallery/forms.py

```diff
@@ -159,6 +159,9 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         self.fields["physical"].required = False
+        self.fields["physical"].label = _(
+            "Changer l'image (attention: cela ne changera pas l'image là où vous l'avez déjà utilisée; un nouvel identifiant sera attribué à la nouvelle image)"
```

Comment:
```suggestion
            "Changer l'image (attention : cela ne changera pas l'image là où vous l'avez déjà utilisée ; un nouvel identifiant sera attribué à la nouvelle image)"
```

Encore deux petits détails typographiques.

Reply:
Corrigé.